server/models/User.js

```diff
@@ -4,6 +4,14 @@ import dbConnection from "./../db/mongoose";
 
 const userSchema = mongoose.Schema(
   {
+    firstName: {
+      type: String,
+      required: "First name is required"
+    },
+    lastName: {
+      type: String,
+      required: "Last name is required"
+    },
```

Comment:
I noticed we had a profile model, should we be using that for this information? You'd probably need to make a profile with the same id as the user kind of thing

Reply:
I agree, we need to make sure that the profile created with the user model will have some way to link them, most likely the id, so that the user data can be used to retrieve the profile data.

Reply:
I'll update the profile model to connect with the user model.